bug: invalid memory address or nil pointer dereference on invalid Cloudformation IAM Policy YAML

**Describe the bug**
Trivy scan throws `invalid memory address or nil pointer dereference` when scanning a Cloudformation template with invalid IAM Policy YAML.
**To Reproduce**
Steps to reproduce the behavior:
1. copy stack.yml (see Example Code) to a directory 
2. go run cmd/defsec/main.go /path/to/bad/stack/file

-- OR --

1. docker run -v `pwd`:/work --workdir /work aquasec/trivy fs --security-checks secret,config,vuln  /work/path/to/bad/stack/file

**Expected behavior**
An error clearly explaining the issue.  e.g.  The Cloudformation YAML/JSON in file blah.yaml is invalid on line x.
**Output of your tfsec command with --debug flag**
```bash
root@818dec18830d:/work# go run cmd/defsec/main.go ./bad
07:58.838531600 system.info                      APP       main
07:58.838782500 system.info                      VERSION   
07:58.838820700 system.info                      OS        linux
07:58.838855300 system.info                      ARCH      amd64
07:58.838888600 system.info                      KERNEL    Linux version 5.10.47-linuxkit (root@buildkitsandbox) (gcc (Alpine 10.2.1_pre1) 10.2.1 20201203, GNU ld (GNU Binutils) 2.35.2) #1 SMP Sat Jul 3 21:51:47 UTC 2021
07:58.838925200 system.info                      TERM      xterm
07:58.838958500 system.info                      SHELL     
07:58.838990900 system.info                      GOVERSION go1.19
07:58.839024900 system.info                      GOROOT    /usr/local/go
07:58.839058200 system.info                      CGO       true
07:58.839092500 system.info                      CPUCOUNT  4
07:58.839125900 system.info                      MAXPROCS  4
07:58.839153900 system.info                      WORKDIR   /work
07:58.839183900 system.info                      UID       0
07:58.839200700 system.info                      EUID      0
07:58.839231400 system.info                      DOCKER    true
07:58.839259100 system.info                      CI        false
07:58.839284100 system.info                      HOSTNAME  818dec18830d
07:58.839317800 system.info                      TEMP      /tmp
07:58.839351400 system.info                      PATHSEP   /
07:58.839384300 system.info                      CMD       /tmp/go-build1324169760/b001/exe/main ./bad
07:58.839458800 terraform.scanner                Scanning [&{/work/bad /work/bad}] at '.'...
07:58.845887600 terraform.scanner                no root modules found
07:58.857126600 cloudformation.scanner.rego      Loaded 4 embedded libraries.
07:58.908009100 cloudformation.scanner.rego      Loaded 116 embedded policies.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x6eb6e44]

goroutine 1 [running]:
github.com/aquasecurity/defsec/pkg/scanners/cloudformation/parser.(*Property).AsRawStrings(0xc002411980)
        /work/pkg/scanners/cloudformation/parser/property.go:149 +0x24
github.com/aquasecurity/defsec/pkg/scanners/cloudformation/parser.(*Property).GetJsonBytes(0xc002411980, {0x0?, 0x0, 0x0?})
        /work/pkg/scanners/cloudformation/parser/property.go:361 +0x32
github.com/aquasecurity/defsec/internal/adapters/cloudformation/aws/iam.getPolicies({{0xc001bd60b0, 0x9}, {0xc000c83380, 0x6, 0x6}, {0x822e751, 0x4}, 0x0, 0xc001bc4b70, 0x0, ...})
        /work/internal/adapters/cloudformation/aws/iam/policy.go:15 +0x25e
github.com/aquasecurity/defsec/internal/adapters/cloudformation/aws/iam.Adapt({{0xc001bd60b0, 0x9}, {0xc000c83380, 0x6, 0x6}, {0x822e751, 0x4}, 0x0, 0xc001bc4b70, 0x0, ...})
        /work/internal/adapters/cloudformation/aws/iam/iam.go:22 +0x885
github.com/aquasecurity/defsec/internal/adapters/cloudformation/aws.Adapt({{0xc001bd60b0, 0x9}, {0xc000c83380, 0x6, 0x6}, {0x822e751, 0x4}, 0x0, 0xc001bc4b70, 0x0, ...})
        /work/internal/adapters/cloudformation/aws/adapt.go:55 +0x5e5
github.com/aquasecurity/defsec/internal/adapters/cloudformation.Adapt(...)
        /work/internal/adapters/cloudformation/adapt.go:12
github.com/aquasecurity/defsec/pkg/scanners/cloudformation.(*Scanner).scanFileContext(0xc000b2d680, {0x8fe2858, 0xc000128008}, 0x7f137c41e180?, 0xc000c833e0, {0x7f137c41e180, 0xc000dee600})
        /work/pkg/scanners/cloudformation/scanner.go:170 +0xd8
github.com/aquasecurity/defsec/pkg/scanners/cloudformation.(*Scanner).ScanFS(0xc000b2d680, {0x8fe2858, 0xc000128008}, {0x7f137c41e180, 0xc000dee600}, {0x822923c?, 0xc001bb9110?})
        /work/pkg/scanners/cloudformation/scanner.go:133 +0x1ba
github.com/aquasecurity/defsec/pkg/scanners/universal.(*Scanner).ScanFS(0xc0007252d0?, {0x8fe2858, 0xc000128008}, {0x7f137c41e180, 0xc000dee600}, {0x822923c, 0x1})
        /work/pkg/scanners/universal/scanner.go:67 +0x133
main.main()
        /work/cmd/defsec/main.go:33 +0x26b
exit status 2
```
**System Info**
 - devsec version: latest
 - Trivy version: latest

**Example Code**
stack.yml
```yaml
Resources:
  MyPolicy:
    Type: AWS::IAM::Policy
    Properties:
      PolicyName: CFNUsers
```
**Additional context**
This is not a terraform project, there is just a stack.yml, which is a cloudformation template in yaml format, which is invalid however trivy bombs out instead of reporting the invalid file.  We want to keep config scanning enabled, but we want to not scan any cloudformation files, as we have a checkov process that take care of that.  If there is a way to disable this behavior while still enabling config scanning, that would be a huge help as well.

Thanks in advance!

--- a/cmd/defsec/main.go
+++ b/cmd/defsec/main.go
@@ -35,14 +35,6 @@ func main() {
 		panic(err)
 	}
 
-	// Execute the API scanners //TODO do we even want this here?
-	apiScanResults, err := s.Scan(context.TODO())
-	if err != nil {
-		panic(err)
-	}
-
-	results = append(results, apiScanResults...)
-
 	if err := formatters.New().WithBaseDir(abs).AsSARIF().Build().Output(results); err != nil {
 		panic(err)
 	}

--- a/internal/adapters/cloudformation/aws/iam/policy.go
+++ b/internal/adapters/cloudformation/aws/iam/policy.go
@@ -9,22 +9,25 @@ import (
 
 func getPolicies(ctx parser.FileContext) (policies []iam.Policy) {
 	for _, policyResource := range ctx.GetResourcesByType("AWS::IAM::Policy") {
-		policyProp := policyResource.GetProperty("PolicyDocument")
-		policyName := policyResource.GetStringProperty("PolicyName")
 
-		doc, err := iamgo.Parse(policyProp.GetJsonBytes())
-		if err != nil {
-			continue
+		policy := iam.Policy{
+			Metadata: policyResource.Metadata(),
+			Name:     policyResource.GetStringProperty("PolicyName"),
+			Document: iam.Document{
+				Metadata: policyResource.Metadata(),
+				Parsed:   iamgo.Document{},
+			},
 		}
 
-		policies = append(policies, iam.Policy{
-			Metadata: policyProp.Metadata(),
-			Name:     policyName,
-			Document: iam.Document{
-				Metadata: policyProp.Metadata(),
-				Parsed:   *doc,
-			},
-		})
+		if policyProp := policyResource.GetProperty("PolicyDocument"); policyProp.IsNotNil() {
+			doc, err := iamgo.Parse(policyProp.GetJsonBytes())
+			if err != nil {
+				continue
+			}
+			policy.Document.Parsed = *doc
+		}
+
+		policies = append(policies, policy)
 	}
 	return policies
 }

